src/core/openthread-core-config.h

```diff
@@ -34,14 +34,10 @@
 #ifndef OPENTHREAD_CORE_CONFIG_H_
 #define OPENTHREAD_CORE_CONFIG_H_
 
-#ifndef OPENTHREAD_CORE_HAVE_PROJECT_SPECIFIC_CONFIG
-#define OPENTHREAD_CORE_HAVE_PROJECT_SPECIFIC_CONFIG 0
-#endif
-
 #define OPENTHREAD_CORE_CONFIG_H_IN
 
-#if OPENTHREAD_CORE_HAVE_PROJECT_SPECIFIC_CONFIG
-#include <openthread-core-project-config.h>
+#ifdef OPENTHREAD_PORJECT_CORE_CONFIG_FILE
```

Comment:
That's unfortunate as this change will break several projects we have going at Nest.